doc/tutorials/visualizing_in_rviz/visualizing_in_rviz.rst

```diff
@@ -112,7 +112,7 @@ After we've constructed and initialized, we now create some closures (function o
 
 .. code-block:: C++
 
-    // Create a closures for visualization
+    // Create closures for visualization
```

Comment:
A bit nitpicky again: Do you mind changing this and the one below in another PR so it is not out of context?

Reply:
Done